### PR TITLE
Fix for #3489

### PR DESCRIPTION
**Pull request by abhishekgupta5**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -117,4 +117,7 @@ org.freedesktop.UDisks2.h
 /po/po.*.js
 /po/po.*.js.gz
 /pkg/storaged/index.min.html.gz
+/.gitignore.swp
+/cockpit-wip.tar.gz
+/srpm-build.Wt9BcW/
```
Comment (mvollmer) [inline .gitignore]: All this looks unrelated.  But yes, our .gitignore is not in a good shape...

Comment (mvollmer) [inline .gitignore]: `srpm-build.Wt9BcW/` looks like a random name.  We should use a pattern.

**Comment by mvollmer** [comment]
I would leave the `.gitignore` cleanup out of this.

**Comment by abhishekgupta5** [comment]
@andreasn Ya, I was confused between 36 or 35. Will correct it

**Comment by abhishekgupta5** [comment]
Why did this test failed ?

**Comment by stefwalter** [comment]
> Why did this test failed ?

Because this is included in the pull request:

```
 +<<<<<<< HEAD
           <textarea id="authorized-keys-text" placeholder="Paste the contents of your public SSH key file here" class="form-control" style="resize:none;margin-bottom:10px"></textarea>
+=======
+          <textarea id="authorized-keys-text" placeholder="Paste the contents of your public ssh key file here" class="form-control" style="resize:none;margin-bottom:10px"></textarea>
+>>>>>>> c0d5e2bda6c3538a62d5bf9d5ea67b8195e2a1d1 
```

**Comment by stefwalter** [comment]
Solved in #3870
